### PR TITLE
 docs: v11 color token updates for date picker component

### PR DESCRIPTION
**Pull request by laurenmrice**
- This PR updates the color tokens on the Style tab for the date picker component.
  - Changed column `Class` --> `Element`

### CODE REVIEW

**Comment by laurenmrice** [comment]
Should the "Today" text for range be `link-01` or `interactive`?

**Review by joshblack — APPROVED** [review]

File: src/pages/components/date-picker/style.mdx [hunk 1]
```diff
@@ -134,16 +138,16 @@ The widths of the date inputs may vary based on the grid and layout.
...
-| `.bx--date-picker__icon`     | padding-right, padding-left              | 16 / 1   | `$spacing-05` |
-| `.today` dot                 | height & width                           | 4 /0.25  | –             |
-| `day`                        | height & width                           | 40 / 2.5 | –             |
+| Element        | Property                                 | px / rem | Spacing token |
+| -------------- | ---------------------------------------- | -------- | ------------- |
+| Calendar       | height                                   | 336 / 21 | –             |
+|                | width                                    | 288 / 18 | –             |
+|                | padding-top, padding-right, padding-left | 4 / 0.25 | `$spacing-02` |
+|                | padding-bottom                           | 8 / 0.5  | `$spacing-03` |
+| Claendar: icon | height, width                            | 16 / 1   | –             |
```
Comment (emyarod) [inline src/pages/components/date-picker/style.mdx:147]:
```suggestion
| Calendar: icon | height, width                            | 16 / 1   | –             |
```

**Review by emyarod — APPROVED** [review]
looks good to me, just one change

File: src/pages/components/date-picker/style.mdx (hunk 1 above)
Comment (laurenmrice) [reply src/pages/components/date-picker/style.mdx:147]: good catch!

**Review by aagonzales — APPROVED** [review]
Looks good!

**Pull request merged**